locations/spiders/auchan_ru.py

```diff
@@ -0,0 +1,38 @@
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_WEEKDAY, DAYS_WEEKEND, OpeningHours
+from locations.items import Feature
+
+
+class AuchanRUSpider(scrapy.Spider):
+    name = "auchan_ru"
+    start_urls = ["https://www.auchan.ru/v1/shops/external-list"]
+    requires_proxy = "RU"
+
+    def parse(self, response):
+        for poi in response.json():
+            item = DictParser.parse(poi)
+            item["state"] = None  # Region values are not accurate
+            self.parse_hours(item, poi)
+            if item.get("name", "").lower().startswith("атак "):
+                item["brand_wikidata"] = "Q2868738"
```

Comment:
Is this the right id?

Atac

French supermarket chain

Wikidata ID
Q2868738
https://www.wikidata.org/wiki/Q2868738
https://www.wikidata.org/wiki/Special:EntityData/Q2868738.json

Reply:
Actually, yes, parent org = Auchan